integration-testing/build.gradle

```diff
@@ -0,0 +1,89 @@
+
+buildscript {
```

Comment:
How do you intend to run this project? Will you set up another build on TC?

Reply:
I'll add the second step in the build on TC that will depend on `publishToMavenLocal`